### PR TITLE
Add HTML5 tag noreferrer

### PR DESCRIPTION
**Pull request by Eragos**
Add rel="noreferrer" to the links. That hides the referrer per default
on supported user agents.

http://www.w3.org/TR/html5/links.html#link-type-noreferrer

### CODE REVIEW

**Comment by Eragos** [comment]
No I hope i've found everything. Sorry for the many commits.

File: tpl/linklist.html [hunk 1]
```diff
@@ -50,13 +50,13 @@
...
+                <span class="linktitle"><a href="{$redirector}{$value.url}" rel="noreferrer">{$value.title}</a></span>
                 <br>
                 {if="$value.description"}<div class="linkdescription">{$value.description}</div>{/if}
                 {if="!$GLOBALS['config']['HIDE_TIMESTAMPS'] || isLoggedIn()"}
-                    <span class="linkdate" title="Permalink"><a href="?{$value.linkdate|smallHash}">{function="strftime('%c', $value.timestamp)"} - permalink</a> - </span>
+                    <span class="linkdate" title="Permalink"><a href="?{$value.linkdate|smallHash}" rel="noreferrer">{function="strftime('%c', $value.timestamp)"} - permalink</a> - </span>
                 {else}
-                    <span class="linkdate" title="Short link here"><a href="?{$value.linkdate|smallHash}">permalink</a> - </span>
+                    <span class="linkdate" title="Short link here"><a href="?{$value.linkdate|smallHash}" rel="noreferrer">permalink</a> - </span>
                 {/if}
```
Comment (ArthurHoaro) [inline tpl/linklist.html]: Those two are internal links.

**Comment by ArthurHoaro** [comment]
Thanks for your work!

Is there a specific reason you add the tag on permalinks? Also, could you squash these commits?

**Comment by Eragos** [comment]
Yes, I add the noreferrer tag also to the permalink because you can accidentally click on those too with the result to submit the referrer. It's a bit uniformer to use the tag on all possible links.

I'm not so familiar with git/github to merge the commit in one, but I'll look at this :)

**Comment by ArthurHoaro** [comment]
Yes, but permalinks are internal links. I don't really see the point of hidding the referer within Shaarli. We actually use it for link edit redirection (not on this link though).

Thanks for the rebase.

**Comment by Eragos** [comment]
> Yes, but permalinks are internal links. I don't really see the point of hidding the referer within Shaarli. We actually use it for link edit redirection (not on this link though).

Yes, this is right. ~~They point to Shaarli and being redirected to the original url. I've tested that a few times. Without noreferrer your referrer is visible to the webserver. With noreferrer not, even Shaarli is redirector in that case.~~

Okay, I see my mistake :) In my setup, I already redirect the permalink instead to show it in the linklist. If #330 will be done, it's necessary to add noreferrer. At the moment you'll right :D

**Comment by Eragos** [comment]
Wrong button :-/

**Comment by Eragos** [comment]
This one should not attached to this pull request. Somehow I have to learn a lot about github.

**Comment by ArthurHoaro** [comment]
In this case, the `noreferrer` tag on permalinks should be in a PR related to #330, not in this one. Could you remove it?

---

Yeah, your PR is on your master while it should be on another branch. No big deal, but you can't commit anything else on your master while it's open.

**Comment by ArthurHoaro** [comment]
Closing this due to lack of feedback.

**Pull request closed without merging**